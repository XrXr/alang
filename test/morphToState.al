main :: proc () {
	a := 1
	b := 2
	c := 3
	d := 4
	e := 5
	f := 6
	g := 7
	h := 8
	i := 9
	j := 10
	k := 11
	l := 12
	m := 13
	n := 14
	stack1 := 1111111
	stack2 := 2222222
	l = 999999
	if false {
		l := n / k
		c += m
		c += n
	}
	print_int(a)
	print_int(b)
	print_int(c)
	print_int(d)
	print_int(e)
	print_int(f)
	print_int(g)
	print_int(h)
	print_int(i)
	print_int(j)
	print_int(k)
	print_int(l)
	print_int(m)
	print_int(n)
	print_int(stack1)
	print_int(stack2)
}
