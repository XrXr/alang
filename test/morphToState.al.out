1
2
3
4
5
6
7
8
9
10
11
999999
13
14
1111111
2222222
